email-alert-service/config/deploy.rb

```diff
@@ -17,7 +17,21 @@
       run "sudo initctl restart email-alert-service-unpublishing-queue-consumer-procfile-worker || "\
           "sudo initctl start email-alert-service-unpublishing-queue-consumer-procfile-worker"
     end
+
+    desc "Restart the update subscriber list details minor consumer"
+    task :restart_subscriber_list_details_update_minor_consumer do
+      run "sudo initctl restart subscriber-list-details-update-minor-consumer-procfile-worker || "\
+          "sudo initctl start subscriber-list-details-update-minor-consumer-procfile-worker"
```

Comment:
These service names (and below) have to match the puppet worker names (not the `process_type`s).  So they'll be `email-alert-service-subscriber-list-details-update-major-consumer` and `email-alert-service-subscriber-list-details-update-major-consumer`

Reply:
the `email-alert-service` bit makes sense to be, but why no `procfile-worker` on the end?
How does this one differ from the above?

https://github.com/alphagov/govuk-puppet/blob/main/modules/govuk/manifests/procfile/worker.pp#L63

Reply:
Oh right yes, it does also need the `-procfile-worker` bit too